packages/syntax-core/src/SelectList/SelectList.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { ReactElement, ReactNode, useId } from "react";
+import styles from "./SelectList.module.css";
+import classNames from "classnames";
+import Typography from "../Typography/Typography";
+import SelectOption from "./SelectOption";
+
+const iconSize = {
+  sm: 20,
+  md: 24,
+  lg: 24,
+} as const;
+
+const SelectList = ({
+  children,
+  disabled = false,
+  error = false,
+  errorText,
+  helperText,
+  label,
+  onChange,
+  placeholderText,
+  selectedValue = "",
+  size = "md",
+}: {
+  /**
+   * One or more SelectList.Option components.
+   */
+  children: ReactNode;
+  /**
+   * true if the select dropdown is disabled
+   */
+  disabled?: boolean;
+  /**
+   * true if there is an error with the selection
+   */
+  error?: boolean;
+  /**
+   * Text shown below select box if there is an input error.
+   * Should only have a value if error = true
+   */
+  errorText?: string;
```

Comment:
@nathan-ong could we remove `error` and only use `errorText`? If `errorText` is set, then we know there is an error

Reply:
Yeah, I thought about this. Can it be safely assumed that there will always be an error text upon error?

Reply:
I would think so, but I'm curious if, for example if we had a form with multiple dropdowns and all fields were required, if we would show the error text individually on each dropdown, or if there would just be an error message on the parent that contained these dropdowns, and we would just make these dropdowns red without an error for each of them?